docs/website/data/integrations.yaml

```diff
@@ -1454,6 +1454,21 @@ integrations:
       - https://github.com/StyraInc/graphql-apollo-example
     tutorials:
       - https://www.openpolicyagent.org/docs/graphql-api-authorization/
+      
+gitgat:
```

Comment:
Indentation is wrong here, this doesn't get picked up by the website build logic. Plase indent this like graphql above.

Reply:
@MrBLightning